### PR TITLE
Addition: Instagram recent pics and video downloader

### PR DESCRIPTION
**Pull request by kartikmandhan**
This Instagram images and video downloader works for public accounts and downloads images and videos of that account

### CODE REVIEW

File: comment_on_all_photos/src.py [hunk 1]
```diff
@@ -18,14 +18,14 @@
...
-
+time.sleep(5)
 user_name = driver.find_element_by_name('username')
 password = driver.find_element_by_name('password')
 
 user_name.send_keys(USERNAME)
 password.send_keys(PASSWORD)
 
-login = driver.find_element_by_class_name('_5f5mN')
+login = driver.find_element_by_xpath('//*[@id="loginForm"]/div/div[3]')
```
Comment (yashshah1) [inline comment_on_all_photos/src.py]: This is from another commit, please merge from upstream

File: recent_image_and_pic_downloader/src.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# import selenium,time & requests modules
+import time
+
+import requests
+
+from selenium import webdriver
```
Comment (yashshah1) [inline recent_image_and_pic_downloader/src.py]:
```suggestion
import time
import requests
from selenium import webdriver
```

Avoid unnecessary newlines

**Review by yashshah1 — CHANGES_REQUESTED** [review]
Add all the downloaded videos into a new folder, which can probably be named the time of the script execution

**Review by kartikmandhan — COMMENTED** [review]
i have made all the desired changes

**Review by kartikmandhan — COMMENTED** [review]
directory structure created for video and image files

**Pull request merged**